### PR TITLE
Create 2019-02-16-community-survey-preview.md

### PR DESCRIPTION
**Pull request by MelSumner**
adds a blog post about the community survey preview

### CODE REVIEW

**Comment by mixonic** [comment]
@MelSumner I pushed a commit here fixing the yaml (I think) which was causing the build to fail.

File: source/blog/2019-02-16-community-survey-preview.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+and [EmberConf 2019](https://emberconf.com/) with the release of the fifth
+annual Ember Community Survey, and as always, we want your input before we go
+live!
+
+In 2015, [201 Created](https://www.201-created.com/) began an annual survey of
+the Ember community. Over the past four years, data from the survey has helped
+us measure progress on issues as wide ranging as diversity and browser support.
+In 2017, free text responses were permitted for the first time giving community
+participants a way to provide feedback to the project and project leadership a
+new way to hear their voices. As 201 Created shuts down, it feels like a good
```
Comment (jenweber) [inline source/blog/2019-02-16-community-survey-preview.md]:
```suggestion
new way to hear their voices. It feels like a good
```

File: source/blog/2019-02-16-community-survey-preview.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+We are thrilled to announce that we will be adding to the spring excitement
+surrounding [Ember Octane](https://github.com/emberjs/rfcs/blob/26c4d83fb66568e1087a05818fb39a307ebf8da8/text/0000-roadmap-2018.md)
+and [EmberConf 2019](https://emberconf.com/) with the release of the fifth
+annual Ember Community Survey, and as always, we want your input before we go
+live!
+
+In 2015, [201 Created](https://www.201-created.com/) began an annual survey of
+the Ember community. Over the past four years, data from the survey has helped
+us measure progress on issues as wide ranging as diversity and browser support.
+In 2017, free text responses were permitted for the first time giving community
```
Comment (jenweber) [inline source/blog/2019-02-16-community-survey-preview.md]:
```suggestion
In 2017, free text responses were permitted for the first time, giving community
```

File: source/blog/2019-02-16-community-survey-preview.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+In 2019 the Ember community is more vibrant than ever, with over 3000 members
+having joined the official Discord server (join via the [Ember Community page](https://www.emberjs.com/community/))
+that launched last September. We're eager to hear how people are using Ember
+and what they plan for the future.
+
+We'd like to get as much feedback as possible about the content of the survey
+before we go live. This year we're making an effort to move much of the survey
+process to the [ember-learn/annual-community-survey GitHub repo](https://github.com/ember-learn/annual-community-survey).
+Please review the [the list of 2019 survey questions](https://github.com/ember-learn/annual-community-survey/blob/master/2019/survey-questions.md)
```
Comment (jenweber) [inline source/blog/2019-02-16-community-survey-preview.md]: Move this up to the top if you want maximum participation. Keep it here if you only want a small # of really dedicated people to chime in.

File: source/blog/2019-02-16-community-survey-preview.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+this before February 17th and submit feedback as
+[an issues on that repo](https://github.com/ember-learn/annual-community-survey/issues).
+
+Here are the key dates for this year's survey: 
+
+* 02/16 - 02/17 : Survey question feedback
+* 02/18 - 03/08 : Survey open to the public for responses
+* 03/19 : Results distributed in time for the EmberConf 2019 keynote!
+
+Please join us in `#dev-ember-learning` on Discord if you have any questions
```
Comment (jenweber) [inline source/blog/2019-02-16-community-survey-preview.md]:
```suggestion
Please join us in `#dev-ember-learning` on [Discord](https://emberjs.com/community/) if you have any questions
```

File: source/blog/2019-02-16-community-survey-preview.md (hunk 3 above)
Comment (mixonic) [reply source/blog/2019-02-16-community-survey-preview.md]: This is a good nudge. I try to follow the rule of "put the actionable thing first" myself, but you caught me forgetting it here. 👍

**Pull request merged**